telethon/client/chats.py

```diff
@@ -1145,6 +1145,23 @@ async def kick_participant(
         else:
             raise ValueError('You must pass either a channel or a chat')
 
+    async def get_permissions(
+            self: 'TelegramClient',
+            channel: 'hints.EntityLike',
+            user: 'hints.EntityLike'
+    ) -> 'custom.ParticipantPermissions':
+        channel = await self.get_input_entity(channel)
+        user = await self.get_input_entity(user)
+        if helpers._entity_type(user) != helpers._EntityType.USER:
+            raise ValueError('You must pass a user entity')
+        if helpers._entity_type(channel) != helpers._EntityType.CHANNEL:
+            raise ValueError('You must pass a channel entity')
```

Comment:
It is actually possible to fetch the permissions of people in small group chats, by using [`GetFullChat`](https://tl.telethon.dev/methods/messages/get_full_chat.html). A friendly method should be as complete as possible, so it definitely should handle both channels and chats. After fetching the full chat, the [`participants`](https://tl.telethon.dev/constructors/chat_participants.html) of [`ChatFull`](https://tl.telethon.dev/constructors/chat_full.html) can be inspected.

Because fetching the entire chat is inefficient to consult just a single participant, it should be noted on the documentation. However, users may be okay with that, since they just need a one-time use for a single person.

Reply:
How should handle chat in `permissions.ban_users`?  Chat participant doesnt have info about this permission

Reply:
I haven't checked how small groups work in a while, but it seems there are only three types: normal user, admin and creator. Probably, if it's either admin or creator, all permissions would be `True` (they can do all of them).